app/models/case_log.rb

```diff
@@ -723,4 +700,57 @@ def soft_value_for_period(value)
   def upcase_and_remove_whitespace(string)
     string.present? ? string.upcase.gsub(/\s+/, "") : string
   end
+
+  def housingneeds_present?
+    housingneeds.present?
+  end
+
+  def fully_wheelchair_accessible?
+    housingneeds_type.present? && housingneeds_type.zero?
+  end
+
+  def essential_wheelchair_access?
+    housingneeds_type == 1
+  end
+
+  def level_access_housing?
+    housingneeds_type == 2
+  end
+
+  def other_housingneeds?
+    housingneeds_other == 1
+  end
+
+  def has_housingneeds?
+    housingneeds == 1
+  end
+
+  def no_housingneeds?
+    housingneeds == 2
+  end
+
+  def unknown_housingneeds?
+    housingneeds == 3
+  end
+
+  def set_housingneeds_fields
+    set_housingneeds_values_to_nil
```

Comment:
Name seems a bit misleading if we're setting to 0 rather than nil?

Reply:
Changed that a bit